python/src/worker/config/__init__.py

```diff
@@ -1,24 +1,22 @@
 import os
 import types
+import re
+from logging import info
```

Comment:
you can remove this import, it is not being used

Reply:
Fixed.